Frontend/ui-driver/src/Screens/Types.purs

```diff
@@ -1339,7 +1352,14 @@ type HomeScreenProps =  {
   showReferNowPopUp :: Boolean,
   showAddUPIPopUp :: Boolean,
   showVerifyUPIPopUp :: Boolean,
-  chatServiceKilled :: Boolean
+  chatServiceKilled :: Boolean,
+  countVisibility :: Boolean,
+  checkUpcomingRide :: Boolean,
+  bannerVisibility :: Boolean,
+  isPillClickable ::  Boolean,
+  pillShimmerVisibility :: Boolean,
```

Comment:
create an object for these two and countVisibility also as they are referring to same object.